libsoftwarecontainer/component-test/softwarecontainerlib_componenttest.cpp

```diff
@@ -487,7 +501,7 @@ TEST_F(SoftwareContainerApp, TestUnixSocket) {
 
                 close(fd2);
 
-                return done == 1 ? EXISTENT : NON_EXISTENT;
+                return done == 1 ? SUCCESS : FAILURE;
```

Comment:
it would be better to map numbers to constexpr as well. 
Also `CONSTEXPRDONE == done` is prefferable